File: problems/91/Main.cpp
#include <iostream>
#include <cstdio>
#include <algorithm>
#include <cstring>
#include <string>
#include <cctype>
#include <stack>
#include <queue>
#include <list>
#include <vector>
#include <map>
#include <sstream>
#include <cmath>
#include <bitset>
#include <utility>
#include <set>
#include <numeric>
#include <ctime>
#include <math.h>

using namespace std;

class Solution {
	public:
		vector<vector<int> > diagonal(vector<vector<int> > &A);
};

//--SPLIT--
//-vector<vector<int> > Solution::diagonal(vector<vector<int> > &A) {
//-}


//--SPLIT--
vector<vector<int> > Solution::diagonal(vector<vector<int> > &A) {
	return A;
}

//--SPLIT--
int main() {
	int testcases;
	cin >> testcases;
	for (int t = 0; t < testcases; t++) {
		int n; cin >> n;
		vector<vector<int> > A(n, std::vector<int>(n));
		for (int i = 0; i < n; i++) {
			for (int j = 0; j < n; j++) {
				cin >> A[i][j];
			}
		}
		Solution solution;
		vector<vector<int> > result = solution.diagonal(A);
		for (int j = 0; j < result.size(); j++) {
			vector<int> v = result[j];
			for (int k = 0; k < v.size(); k++) {
				cout << v[k]  << " ";
			}
			cout << "\n";
		}
	}
	return 0;
}

File: problems/91/Main.py
from __future__ import print_function
import math


#--SPLIT--
#-class Solution:
	# @param a : list of list of integers
	# @return a list of list of integers
#-	def diagonal(self, a):




#--SPLIT--
class Solution:
	# @param a : list of list of integers
	# @return a list of list of integers
	def diagonal(self, a):
		return a



#--SPLIT--

if __name__ == "__main__":
	test_cases = int(raw_input())
	for i in range(test_cases):
		n = int(raw_input())
		a = []
		for i in range(n):
			a.append([int(l) for l in raw_input().split(" ")])
		solution = Solution()
		result = solution.diagonal(a)
		for r in result:
			for x in r:
				print(str(x) + " ", end="")
			print("")
